### PR TITLE
app-text/evince: Version bump to 44_rc BETA

### PR DESCRIPTION
**Pull request by flint2**

### CODE REVIEW

File: app-text/evince/evince-44_rc.ebuild [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+
+		-Dcomics=enabled
+		$(meson_feature djvu)
+		$(meson_feature dvi)
+		-Dpdf=enabled
+		$(meson_feature postscript ps)
+		$(meson_feature tiff)
+		$(meson_feature xps)
+
+		-Duser_doc=true
```
Comment (mattst88) [inline app-text/evince/evince-44_rc.ebuild:99]: You removed gtk_doc here, I believe mistakenly.

Why did you remove the USE flag? Even if there's a reason to remove the USE flag, you should explicitly enable or disable the config option.

**Review by mattst88 — CHANGES_REQUESTED** [review]
````diff
 * QA Notice: The ebuild is installing to one or more unexpected paths:
 * 
 *   /usr/share/doc/libevdocument
 *   /usr/share/doc/libevview
 * 
 * Please fix the ebuild to use correct FHS/Gentoo policy paths.
 * 
 * QA Notice: This package seems to install gtk-doc documentation into the
 * following location(s):
 * 
 *   /usr/share/doc/libevview
 *   /usr/share/doc/libevdocument
 * 
 * gtk-doc documentation must always be installed into /usr/share/gtk-doc.
 * For more details, please see the GNOME team policy page:
 * https://wiki.gentoo.org/wiki/Project:GNOME/Gnome_Team_Ebuild_Policies#Handling_gtk-doc_documentation
````

Should fix this while we're here. Bug: https://bugs.gentoo.org/866001

File: app-text/evince/evince-44_rc.ebuild (hunk 1 above)
Comment (flint2) [reply app-text/evince/evince-44_rc.ebuild:106]: Yes, misinterpreted the change to depend gtk-doc on instrospection, reverted and added IUSE dependency

**Comment by flint2** [comment]
> ```diff
>  * QA Notice: The ebuild is installing to one or more unexpected paths:
>  * 
>  *   /usr/share/doc/libevdocument
>  *   /usr/share/doc/libevview
>  * 
>  * Please fix the ebuild to use correct FHS/Gentoo policy paths.
>  * 
>  * QA Notice: This package seems to install gtk-doc documentation into the
>  * following location(s):
>  * 
>  *   /usr/share/doc/libevview
>  *   /usr/share/doc/libevdocument
>  * 
>  * gtk-doc documentation must always be installed into /usr/share/gtk-doc.
>  * For more details, please see the GNOME team policy page:
>  * https://wiki.gentoo.org/wiki/Project:GNOME/Gnome_Team_Ebuild_Policies#Handling_gtk-doc_documentation
> ```
> 
> Should fix this while we're here. Bug: https://bugs.gentoo.org/866001

Fixed in https://github.com/gentoo/gentoo/pull/30072/commits/305d06f393401971e9c89ea90ac885f4a6b59346

File: app-text/evince/evince-44_rc.ebuild [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+	meson_src_configure
+}
+
+src_install() {
+	meson_src_install
+
+	if use gtk-doc; then
+	   mkdir -p "${ED}"/usr/share/gtk-doc/html/ || die
+	   mv "${ED}"/usr/share/doc/{libevdocument,libevview} "${ED}"/usr/share/gtk-doc/html/ || die
+	fi
```
Comment (flint2) [inline app-text/evince/evince-44_rc.ebuild:128]: Fixing https://bugs.gentoo.org/866001
